src/static.js

```diff
@@ -1,30 +0,0 @@
-const pCss = `color: #9fa5a9; line-height: 20px!important; text-align: center; font-family: Graphik,sans-serif;`
```

Comment:
Why have you deleted the optional 2FA logic?

Reply:
the file existed but wasn't being used

Reply:
Let's keep this code somewhere if we reinstall the mobile phone 2Fa

Reply:
it's in commit 60c0f2b no need to keep the file around